NCATS_Data_Dictionary.md

```diff
@@ -734,13 +735,20 @@ sent to the target
   * 0: Does not evoke a sense of greed
   * 1: Email appeals to greed and monetary gain are within the theme of the email
 
+### user_reports Collection ###
+* `assessment` [string]: PCA assessment identifier that this user_report is associated with
+* `campaign` [string]: PCA campaign identifier that this user_report is associated with  
+* `customer` [string]: PCA customer identifier that this user_report is associated with
+* `first_report` [ISO date]: Timestamp when the first user click is reported for a campaign
+* `total_num_reports` [integer]: The total number of clicks reported for a campaign
+
 ### users Collection ###
 
 * `_id` [ObjectId]: PCA ID of this user
 * `customer` [string]: PCA customer identifier that this user is associated with
 * `customer_defined_labels` [dictionary]: Customer-defined text labels for
 grouping users for statistical purposes
-
+  
```

Comment:
What happened here?  Looks like some stray whitespace that should be deleted- this should just be an empty line.

Reply:
Take a look at 9e4946ccfe13fba6f361af63b58caf4ab06244c8, I swatted that stray whitespace

Reply:
Thanks!